### ISSUE TITLE
Last rows dissappear for the first print.

### ISSUE THREAD

**Issue by hamzamon** [andriyoganp/blue_print_pos#30]
When printing a long receipt on the first print after the launch app, the last rows will disappear and then be printed correctly.

**Comment by andriyoganp** [comment]
It happen only in first or next print as well?. You can use method `.addSpacer` or use `feedCount` when print execution

**Comment by hamzamon** [comment]
yes, only for first print until close app.

**Comment by iandis** [comment]
@hamzamon Yes it's because on Android, the `WebView` needs to "warm up" for the first launch. So you need to print blank on first print. However it's working perfectly on iOS without the need to print blank on first print.

**Comment by kechkibet** [comment]
@iandis is this a webview issue, can you link to the issue?

**Comment by kechkibet** [comment]
cant we use flutter widget to image instead of webview?
